src/commands/publish-helm.yml

```diff
@@ -59,7 +64,7 @@ steps:
         git remote rm origin
         git remote add origin "https://${GIT_USER}:${GIT_TOKEN}@${GIT_HOST}/${CHART_ORG}/${CHART_REPO}.git"
         git fetch
-        git checkout gh-pages
+        git checkout <<parameters.git-branch>>
```

Comment:
To keep in line with how other variables are used, this should be added to an env variable called `$GIT_BRANCH`

Reply:
Fixed